etc/custom.properties

```diff
@@ -140,6 +140,23 @@ org.opencastproject.workspace.cleanup.period=86400
 org.opencastproject.workspace.cleanup.max.age=2592000
 
 
+######### Working File Repository Cleanup #########
```

Comment:
Why make this a system-wide configuration and not a service configuration?

Reply:
Well I thought cleaning up the working file repository is essencialy the same as cleaning up the workspace. The Workspace Cleanup is configured in the custom.properties file, thus I placed the configuration for the WFR cleanup just below in the same file. Should I change the location?